src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Config/EventHubOptions.cs

```diff
@@ -251,7 +226,8 @@ internal EventProcessorHost GetEventProcessorHost(string eventHubName, string co
                 var storageConnectionString = creds.StorageConnectionString;
                 if (storageConnectionString == null)
                 {
-                    storageConnectionString = _defaultStorageString;
+                    string defaultStorageString = config.GetWebJobsConnectionString(ConnectionStringNames.Storage);
+                    storageConnectionString = defaultStorageString;
```

Comment:
Can `StorageConnectionString` be elevated to a property on `EventHubOptions` that we set during our configuration? That would allow someone to override this with `Configure<EventHubOptions>()` like they can for other options objects.

It'd also let you remove `IConfiguration` from being passed in here.